cli/src/libindy/mod.rs

```diff
@@ -83,6 +83,9 @@ pub enum ErrorCode
     // Trying to open wallet that was opened already
     WalletAlreadyOpenedError = 206,
 
+    // Attempt to open encrypted wallet with invalid credentials
+    WalletSecurityError = 207,
```

Comment:
@vimmerru `rusqlcipher::ErrorCode::NotADatabase` is returned by `rusqlcipher` in case of incorrect key. May be we should introduce slightly different error code, like `WalletFailAccessToDatabase` ?

Reply:
For this case we can't authenticate. I more like WalletSecurityError approach or WalletInvalidCredentials.

Reply:
But `rusqlcipher` doesn't provide ability to distinct security error and invalid database file

Reply:
@Artemkaaas after discussion we suggest to use `WalletAccessFailed`